### DISCUSSION TITLE
LADSPA and/or other plugin support

### DISCUSSION THREAD

**Discussion by dakhubgit** [jamulussoftware/jamulus#1258] [Ideas]
Jamulus offers a reverb feature in its UI.  That could certainly be provided rather well using LADSPA or other well-known plugin architectures.  The reason it occured to me is that I have a large diaphragm condensor that tends to end up rather harsh on accordion, and being able to use a common-sense IIR EQ would allow to tamper this a bit in the digital realm.  Also it would mean that maintaining the reverb functionality inside of Jamulus itself would no longer be necessary: it does seem like a somewhat quirky addition, and making quirks the choice of the user rather than of the developer team seems to offer more choice.

Now I could probably do something like that with a plugin rack in Jack before even routing the input into Jamulus, but I've lent out my microphones to users running Windows where this is not as easy an option as on GNU/Linux.

**Comment by 38github** [comment]
Wouldn't for example VST/VST3 be more useful on Windows compared to LADSPA? On Linux the addition of LADSPA and/or LV2 would be great in my opinion.

**Comment by gilgongo** [comment]
See also https://github.com/corrados/jamulus/issues/82 (so is this a dupe?)

**Comment by fleutot** [comment]
@38github My spontaneous reaction is that integrating this to Jamulus goes against the Unix philosophy: have tools that do one things well, and combine them. I know Windows doesn't give a damn. I use jack-rack with Jamulus (raysession makes the jack connections quick and easy). Why does signal pre-conditioning need to be integrated to Jamulus? I've heard Windows people route through DAWs to that effect (pun!)?

@dakhubgit Are you asking for an interface, inside Jamulus, to effects outside Jamulus? I mean... implementing interfaces to the hundredish LADSPA effects would be daunting, and quite off topic? Maybe I'm missing something.

**Comment by gilgongo** [comment]
Moving to discussion until we can get an agreement on action.
